generators/entity/index.js

```diff
@@ -1000,6 +1040,50 @@ module.exports = EntityGenerator.extend({
 
     prompting: {
         /* pre entity hook needs to be written here */
+        askForMicroserviceJson: function(){
+            if(this.applicationType != 'gateway' || this.useConfigurationFile) {
+                return;
+            }
+
+            var cb = this.async();
+
+            var prompts = [
+                {
+                    type: 'confirm',
+                    name: 'useMicroserviceJson',
+                    message: 'Do you want to generate this entity from an existing microservice?',
+                    default: true
+                },
+                {
+                    when: function(response) {
+                        return response.useMicroserviceJson == true;
+                    },
+                    type: 'input',
+                    name: 'microservicePath',
+                    message: 'Enter the absolute path to the microservice root directory:',
```

Comment:
The problem I got by using relative paths is that as we are in the jhipster generator folder, the relative path is from the jhipster generator folder and not from the gateway folder... That's why I precisely ask to the user the absolute path.
If you know a way to handle relative path from where the generator is launched let me know.